build/build-image/cross/Dockerfile

```diff
@@ -38,3 +38,21 @@ RUN apt-get install -y g++ && apt-get clean && rm -rf /var/lib/apt/lists/* &&\
     rm -rf protobuf-3.0.0-beta-2 &&\
     protoc --version
 
+# (set an explicit GOARM of 5 for maximum compatibility)
+ENV GOARM=5 GOOS=linux GOARCH=amd64
+
+# work around 64MB tmpfs size in Docker 1.6
+ENV TMPDIR /tmp.k8s
+
+# Get the code coverage tool and godep
+RUN mkdir $TMPDIR && \
+    go get golang.org/x/tools/cmd/cover github.com/tools/godep
+
+# Download and symlink etcd.  We need this for our integration tests.
+RUN export ETCD_VERSION=v2.2.1; \
+    mkdir -p /usr/local/src/etcd &&\
+    cd /usr/local/src/etcd &&\
```

Comment:
I would combine these three lines to
```
curl -sSL https://github.com/coreos/etcd/releases/download/$ETCD_VERSION/etcd-$ETCD_VERSION-linux-amd64.tar.gz | tar xz -C /usr/local/src/etcd
```
It has the advantage that the `.tar.gz` isn't left in the fs and taking up space